benchmarks/http_propagation_inject/config.yaml

```diff
@@ -15,3 +16,25 @@ with_priority_and_origin:
   sampling_priority: "1"
   dd_origin: "synthetics"
 
+with_tags:
+  <<: *defaults
+  meta: |
+    {"_dd.p.test": "value", "_dd.p.sample": "value", "will.be": "skipped"}
```

Comment:
this should be a valid tag, since we are not going to blindly inject all `_dd.p.*` tags.